docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ WORKDIR $DOCKER_WORKDIR
 COPY pyproject.toml setup.cfg ./
 COPY src/$DOCKER_NAMESPACE_PACKAGE_DIR/__init__.py src/$DOCKER_NAMESPACE_PACKAGE_DIR/
 RUN pip install . --no-binary rasterio \
-    && rm -r /opt/conda/lib/python$PYTHON_VERSION/site-packages/$DOCKER_NAMESPACE_PACKAGE_DIR
+    && rm -r /opt/conda/lib/python$PYTHON_VERSION/site-packages/$DOCKER_NAMESPACE_PACKAGE_DIR || true
```

Comment:
workaround for https://github.com/stactools-packages/sentinel1/issues/31

Reply:
Update the `$PYTHON_VERSION` to 3.10 instead of this (e.g. https://github.com/stactools-packages/noaa-cdr/commit/bef95fad930cb08732ad3dbb49362d3335821aeb).

Alternatively, you could ape https://github.com/stactools-packages/template/pull/75 and rework the Docker setup altogether.